src/item.cpp

```diff
@@ -3250,36 +3250,89 @@ void item::armor_protection_info( std::vector<iteminfo> &info, const iteminfo_qu
 
     if( parts->test( iteminfo_parts::ARMOR_PROTECTION ) ) {
         const std::string space = "  ";
+        const std::string percent = "%";
         // NOLINTNEXTLINE(cata-translate-string-literal)
         std::string bp_cat = string_format( "{%s}ARMOR", bp_name );
 
         bool printed_any = false;
 
-        info.emplace_back( "DESCRIPTION", string_format( "<bold>%s%s</bold>:", bp_desc,
-                           _( "Protection" ) ) );
-        if( bash_resist( false, bp ) >= 1 ) {
-            info.emplace_back( bp_cat, string_format( "%s%s", space, _( "Bash: " ) ), "",
-                               iteminfo::is_decimal, bash_resist( false, bp ) );
+        // gather all the protection data
+        // the rolls are basically a perfect hit for protection and a
+        // worst possible
+        resistances worst_res = resistances( *this, false, 99, bp );
+        resistances best_res = resistances( *this, false, 0, bp );
+
+        const armor_portion_data *portion = portion_for_bodypart( bp );
+        int percent_best = portion->best_protection_chance;
+        int percent_worst = portion->worst_protection_chance;
+
+        if( percent_worst > 0 ) {
+            info.emplace_back( "DESCRIPTION",
+                               string_format( "<bold>%s%s</bold>: <bad>%d%s</bad>, <good>%d%s</good>", bp_desc, _( "Protection" ),
+                                              percent_worst, percent, percent_best, percent ) );
```

Comment:
You can use `%%` in the format string to denote percentage symbol.

Reply:
thank you I could not figure it out